docs/auth/README.adoc

```diff
@@ -13,3 +15,149 @@ dependencies {
 
 == Usage
 
+To use the Auth SDK you'll first need to:
+
+* Import the Core module. See link:../core/README.adoc[the Core documentation].
```

Comment:
I think initial idea for core was to be independent, but now core is part of auth:
https://github.com/aerogear/aerogear-android-sdk/blob/master/auth/build.gradle#L30

Import is not needed - in fact it may cause issues.

Reply:
I'm not sure I understand, according to https://github.com/aerogear/aerogear-android-sdk/pull/52/files#diff-27473e51ddc3d43d92d09cbd9da9e521R58 you'll need to import core and auth in order to use auth in your app? @secondsun Is that correct?

Reply:
@aidenkeating Yeah - I was expecting SDK to require core,
 but it seems that core will be available in the mobile app anyway.

@secondsun we could correct some cases in documentation.